Block/ProductLabel/ProductLabel.php

```diff
@@ -210,9 +210,13 @@ public function getIdentities(): array
     {
         $identities = [];
 
-        /** @var IdentityInterface $product */
+        /** @var IdentityInterface|null $product */
         $product = $this->getProduct();
 
+        if ($product === null) {
+            return [\Smile\ProductLabel\Model\ProductLabel::CACHE_TAG];
```

Comment:
Missing class import `use ...`

Reply:
My bad, I blindly copy/paste the lines that were here before